rerun_py/pyproject.toml

```diff
@@ -47,6 +47,8 @@ rerun = "rerun_bindings:main"
 
 # target-version = "py38"  # inferred from requires-python, see https://beta.ruff.rs/docs/settings/#target-version
 
+unsafe-fixes = true
```

Comment:
can you add a comment explaining what this is, so the reader (me) don't have to google it? 🙏